specifications/catalog/example/dcat.distribution.example.json

```diff
@@ -0,0 +1,84 @@
+{
+    "@context":  "https://w3id.org/idsa/v5/context.json",
+    "@id": "http://provider.com/catalog1/dataset1/",
+    "@type": "dcat:Dataset",
+    "dct:title" : "IDS Dataset #1",
+    "dct:description": [ {
+        "@language": "en",
+        "@value": "This is the number 1 data asset of the IDS. The consumer must be a TRUST_PLUS certfified connector."
+    }, {
+        "@language": "de",
+        "@value": "Dies ist das Nummer 1 Data Asset im IDS. Der Consumer muss ein TRUST_PLUS zertifizierter Connetor sein."
+    } ],
+    "dcat:keyword": [
+        "IDS",
+        "data asset",
+        "dcat dataset"
+    ],
+    "dcat:distribution": [ {
+        "@type": "dcat:Distribution",
+        "dct:title" : "Distribution#1",
+        "dct:description": "This is a distribution which provides the data via a HTTP_REST dataplane.",
+        "dcat:keyword": [
+            "dataplane",
+            "public",
+            "distribution"
+        ],
+        "ids:dataPlaneKind": {
```

Comment:
Thinking about this, the HTTP_REST is not strictly speaking a "data plane" kind. We may be using "data plane" to signify too many things. In my mind, a data plane is some type of infratstructure that is either controlled by a provider or consumer and is responsible for transferring data from a source to a destination. As such, it is an implementation detail. HTTP_REST defines a transport protocol stack for transferring data and must be understood by both the consumer and provider. A data plane may support one or more transport protocols. With that in mind, can we rename this, "transportType" or "transportKind"? Use of "type" may be more common.

Reply:
Yes, the dataplane term does not really fit at the level needed here. I like "transportType", and the value shall be a "binding" or some other kind of transport specification. Not sure I have the right word yet. 

About the plurals: JSON-LD/RDF have the convention to not use plural properties (one does not really know - and care - if an array or individual is at the value position). That's different in the JSON world of course. But as we are defining a JSON-LD namespace, I am arguing for the JSON-LD/RDF way.